tests/components/figure.test.ts

```diff
@@ -40,8 +40,8 @@ function createFigure(
     id: "someuuid",
     x: 1,
     y: 1,
-    height: 100,
-    width: 100,
+    height: 200,
+    width: 200,
```

Comment:
use the `figureParameters` argument instead of modifying the default value and having to change a random test that has nothing to do with what you're fixing...